src/integrationTest/resources/tests/arm/service/testGenerateAnnotationArchiveRecord/expectedResponse.a360

```diff
@@ -6,7 +6,7 @@
     "publisher": "DARTS",
     "region": "GBR",
     "recordDate": "2023-09-23T13:00:00Z",
-    "event_date": "<UPLOADED_DATE_TIME>",
+    "event_date": "<EVENT_DT>",
```

Comment:
seen a few PR's recently where we've ditched the shortened _DT for longer _DATE_TIME to keep consistency with other fields. maybe consider this here

Reply:
Done